spec/features/user_adds_a_new_round_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.